utils/tsh.c

```diff
@@ -65,6 +65,40 @@ static char **_tsh_split_cmd(char *cmd)
 	return ts;
 }
 
+pid_t bgpids[128];
```

Comment:
The size (128) is used in both of the functions below, it should be a macro

Reply:
done